Sources/Unbox.swift

```diff
@@ -95,20 +96,20 @@ public enum UnboxValueError: ErrorType, CustomStringConvertible {
         }
     }
     
-    /// Thrown when a required key was missing in an unboxed dictionary. Contains the missing key.
+    /// Thrown when a required key/value was missing in an unboxed dictionary. Contains the missing key.
     case MissingValueForKey(String)
     /// Thrown when a required key contained an invalid value in an unboxed dictionary. Contains the invalid
     /// key and a description of the invalid data.
     case InvalidValue(String, String)
 }
 
-/// Enum describing errors that can occur during unboxing. Use the throwing functions to receive any errors.
+/// Enum describing errors that can occur during unboxing
 public enum UnboxError: ErrorType, CustomStringConvertible {
     public var description: String {
         let baseDescription = "[Unbox error] "
         
         switch self {
-        case .InvalidInput(let errors):
+        case .InvalidValueErrors(let errors):
```

Comment:
Or how about `InvalidValues`? Since the word `Error` is already in the enum name?